contracts/GSN/IRelayHub.sol

```diff
@@ -162,27 +225,42 @@ contract IRelayHub {
         RecipientBalanceChanged  // The transaction was relayed and reverted due to the recipient's balance changing
     }
 
-    // Returns how much gas should be forwarded to a call to relayCall, in order to relay a transaction that will spend
-    // up to relayedCallStipend gas.
+    /**
+     * @dev Returns how much gas should be forwarded to a call to {relayCall}, in order to relay a transaction that will
+     * spend up to `relayedCallStipend` gas.
+     */
     function requiredGas(uint256 relayedCallStipend) public view returns (uint256);
 
-    // Returns the maximum recipient charge, given the amount of gas forwarded, gas price and relay fee.
+    /**
+     * @dev Returns the maximum recipient charge, given the amount of gas forwarded, gas price and relay fee.
+     */
     function maxPossibleCharge(uint256 relayedCallStipend, uint256 gasPrice, uint256 transactionFee) public view returns (uint256);
 
     // Relay penalization. Any account can penalize relays, removing them from the system immediately, and rewarding the
```

Comment:
Feels like we need a section header. 
Suggest having `Relay penalization` on it's own line, so that it looks like a section rather than a hanging comment.
```suggestion
     // Relay penalization. 
     // Any account can penalize relays, removing them from the system immediately, and rewarding the
```